akka-osgi/build.sbt

```diff
@@ -1,7 +1,7 @@
-import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
+import akka.{ AkkaBuild, Dependencies, Formatting, OSGi, Dist }
 
 AkkaBuild.defaultSettings
-AkkaBuild.dontPublishSettings
+Dist.includeInDist := false
```

Comment:
I know there was the ticket but it didn't explain *why* it shouldn't be part of the dist. Do you know it?